lib/index.js

```diff
@@ -45,18 +45,19 @@
  * The main interface is the {@link analyze} function.
  * @module escope
  */
-"use strict";
-
 /* eslint no-underscore-dangle: ["error", { "allow": ["__currentScope"] }] */
 
-const assert = require("assert");
+import { readFileSync } from "fs";
+import assert from "assert";
+
+import ScopeManager from "./scope-manager.js";
+import Referencer from "./referencer.js";
+import Reference from "./reference.js";
+import Variable from "./variable.js";
 
-const ScopeManager = require("./scope-manager");
-const Referencer = require("./referencer");
-const Reference = require("./reference");
-const Variable = require("./variable");
-const Scope = require("./scope").Scope;
```

Comment:
Just double checking - it looks like Scope was imported just to be exported again. Is it actually safe to remove it completely?

Reply:
It appears I had missed that only `Scope` was being re-exported, so I can return to that basic approach if you wish (i.e., to stop re-exporting the other `Scope` subclasses and only re-export `Scope`).

As to whether `Scope` itself can be dropped, I don't personally know enough on how the package is being used, so I guess that may be a question for others here. But I can mention that the current `exports` would not allow access to the `scope.js` submodule.

Reply:
`Scope` is used in `@angular-eslint/template-parser` [here](https://github.com/angular-eslint/angular-eslint/blob/a4beae803e48197b9897eb9a9742186dc18aa4b5/packages/template-parser/src/index.ts#L5)

Reply:
Subclasses are imported in `@typescript-eslint/experimental-utils` [here](https://github.com/typescript-eslint/typescript-eslint/blob/b1d44497ddf8ef4615c727fe4241e54f7f1aba65/packages/experimental-utils/src/ts-eslint-scope/Scope.ts#L3-L15) (directly from `lib/scope` ). I'm not sure if they're actually used or it's just for types.

Reply:
It seems like we should leave Scope as an export here.

Reply:
Re: leaving Scope, how about the subclasses? If you want those, I can either re-export them too as I am now or we'd have to expand `exports` to allow access to the Scope file.

Reply:
> or we'd have to expand `exports` to allow access to the Scope file.

If we decide to export some modules as subpaths (I guess we won't, but just in case), then we should also build their cjs equivalents in `dist/`.

Reply:
I don't think we need to provide any subpaths at this point (given latest responses from typescript-eslint and Webpack).

@brettz9 given that these are internal-only, my vote is to just export the same things we've already been exporting so we aren't changing the overall structure too much. If we aren't already exporting subclasses, I don't think there's a good reason to do so.

Reply:
Ok, sure. I've added a commit so it is back to just exporting `Scope`.